bench/bench-bytestring.cabal

```diff
@@ -179,18 +179,22 @@ benchmark bench-builder-boundscheck
         Data.ByteString.Unsafe
         Paths_bench_bytestring
 
---executable bench-builder-csv
---  hs-source-dirs:   .. .
---  main-is:          CSV.hs
---  build-depends:    base, deepseq, ghc-prim,
---                    text, binary,
---                    criterion
---  c-sources:        ../cbits/fpstring.c
---                    ../cbits/itoa.c
---  include-dirs:     ../include
---  ghc-options:      -O2
---                    -fmax-simplifier-iterations=10
---                    -fdicts-cheap
---                    -fspec-constr-count=6
---  default-language: Haskell98
+executable bench-builder-csv
+  hs-source-dirs:   .. .
+  main-is:          CSV.hs
+  build-depends:    base, deepseq, ghc-prim,
+                    text, binary,
```

Comment:
Is the import of `binary` still necessary?

Reply:
Good point